test/specs/events/create.js

```diff
@@ -129,7 +129,7 @@ describe('Creating a Future One-off Event', function () {
   });
 
   it('should appear correctly on the "My Events" page', function () {
-    var expectedDate = moment(eventDetails.date).format('Do MMMM YY') + ', ' +
+    var expectedDate = moment(eventDetails.date).format('Do MMMM Y') + ', ' +
```

Comment:
According to https://github.com/CoderDojo/cp-zen-platform/blob/master/web/public/js/services/event-utils.js#L83 , it's supposed to be YYYY
Tests doesn't pass with YY anymore due to changes to https://github.com/CoderDojo/community-platform/issues/1201
Y seems to be a short version-ish of YYYY https://momentjs.com/docs/#/parsing/string-format/